dask/dataframe/utils.py

```diff
@@ -812,10 +812,16 @@ def meta_series_constructor(like):
 def get_string_dtype():
     """Depending on config setting, we might convert objects to pyarrow strings"""
     return (
-        pd.StringDtype("pyarrow") if config.get("dataframe.convert_string") else object
+        pd.StringDtype("pyarrow")
+        if bool(config.get("dataframe.convert_string"))
+        else object
     )
 
 
 def pyarrow_strings_enabled():
     """Config setting to convert objects to pyarrow strings"""
     return bool(config.get("dataframe.convert_string"))
+
+
+def format_string_dtype():
+    return "string" if pyarrow_strings_enabled() else "object"
```

Comment:
This seems not worth a dedicated utility given it's not super complex and is only used in one place

Reply:
Can we make use of `get_string_dtype` where `format_string_dtype` is currently being used instead?

Reply:
I wanted to do that, but both `str(object)` and `repr(object)` return `<class 'object'>`, when I just need `object`.